docker/bookkeeper/entrypoint.sh

```diff
@@ -61,14 +63,14 @@ echo "wait for zookeeper"
 until ${ZK_HOME}/bin/zkCli.sh -server $ZK_URL ls /; do sleep 2; done
 
 echo "create the zk root"
-${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}
-${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}
-${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}
+# Silence exit codes with "|| :" as the commands can safely fail with a "Node already exists" error
+${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH} || :
+${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME} || :
+${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME} || :
```

Comment:
You sure there is no way to submit three commands through the same session with the CLI, rather than doing one session per command? That should be a simple addition.

Reply:
The [ZooKeeper Create Node API](https://zookeeper.apache.org/doc/r3.4.6/api/org/apache/zookeeper/ZooKeeper.html#create(java.lang.String,%20byte[],%20java.util.List,%20org.apache.zookeeper.CreateMode)) does not allow to create nodes in paths that do not exist. To do this, we would need to wrap it in a function, but I don't think it would add any value to the current solution.

Reply:
That's not what I meant. We should be able to do several create operations over the same session. Each execution of zkCli will create a new session unnecessarily.